roles/setup/tasks/main.yml

```diff
@@ -15,7 +15,7 @@
   ansible.builtin.set_fact:
     additional_python_packages: "{{ additional_python_packages + ['receptorctl'] }}"
   when:
-    - receptor_install_method in ['release', 'local']
+    - receptor_install_method in ['release', 'local', 'container']
```

Comment:
Is this still required? An alias for `receptorctl` will be placed in the later step, so do we still need to install `receptorctl` by `pip`?

Reply:
Great catch! This is likely not needed anymore since the alias was updated